geniza/corpus/templates/corpus/document_detail.html

```diff
@@ -21,6 +21,16 @@ <h1>{{ document.title }}</h1>
         {% comment %}Translators: Date document was first added to the PGP{% endcomment %}
         <dt class="sr-only">{% translate 'First input' %}</dt>
         <dd>{{ document.log_entries.last.action_time.year }}</dd>
+
+        {% if document.editions %}
+        {% comment %}Translators: Editor label{% endcomment %}
```

Comment:
my brain exploded when i realized this was a django i18n comment to go in the `.po` file, because some of the things are that are about to be listed below _are_ translators